src/components/AddTaskContainer.tsx

```diff
@@ -0,0 +1,61 @@
+import {
+  ChangeEvent,
+  Dispatch,
+  FormEvent,
+  SetStateAction,
+  useState,
+} from 'react';
+import { addTask, getTasks } from '../utils/api';
+import { Task } from '../utils/types';
+import AddTaskUI from './AddTaskUI';
+
+interface AddTaskContainerProps {
+  setTasks: Dispatch<SetStateAction<Task[]>>;
+}
+
+function AddTaskContainer({ setTasks }: AddTaskContainerProps) {
+  const [newTask, setNewTask] = useState<string>(''); //tracks new task input
```

Comment:
Nitpick 🪡:  you don't need to put the `<string>`. Typescript will be able to infer the type,

Reply:
removed